README.md

```diff
@@ -420,26 +420,28 @@ it from the [official website](https://golang.org/dl/).
 
 2. Follow the [Getting started](https://doc.qt.io/qt-6/android-getting-started.html) page.
 
-3. Set the `QT_HOST_PATH` environment variable to point to the location of the `androiddeployqt` tool.
+3. Set the `QT_HOST_PATH` environment variable to point to the location of the `androiddeployqt` tool  -- minus the `/bin` suffix i.e. if `$(which androiddeployqt)` is `$HOME/Qt/6.2.4/gcc_64/bin/androiddeployqt`, `QT_HOST_PATH` is `$HOME/Qt/6.2.4/gcc_64/`.
 
-4. Set the `ANDROID_SDK_ROOT` and `ANDROID_NDK_ROOT` (required version: 21.0.6113669) environment variables,
-to point to the Android SDK and NDK installation directories.
+4. Set the `ANDROID_SDK_ROOT` and `ANDROID_NDK_ROOT` environment variables,
+to point to the Android SDK and NDK intallation directories. Required NDK versions: 23.1.7779620 and 21.0.6113669.
```

Comment:
Sanity check from @strseb that both are really required. I was only able to build with both... But maybe it's something in my environment?

Reply:
Uh! Yes, doc-rot. It should only require 23.1.7779620.

Reply:
Ok, so the thing is. When I installed `23.1.7779620` I removed `21.0.6113669` and I that asked me for `21.0.6113669`.

Reply:
Hm, if CI only has the 23 version I'll leave jsut that. Requiring two versions is weird.

Reply:
Maybe QT has something that depends on 21.x? :/